schema.sql

```diff
@@ -7,6 +7,7 @@ CREATE TABLE IF NOT EXISTS `accounts` (
   `lastday` int(10) unsigned NOT NULL DEFAULT '0',
   `email` varchar(255) NOT NULL DEFAULT '',
   `creation` int(11) NOT NULL DEFAULT '0',
+	`secret` char(16) DEFAULT NULL,
```

Comment:
Only a player does not turn on the server can not set up an account on znote aac